LICENSE

```diff
@@ -201,3 +201,8 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
+********************************************************************************
+Apache NetBeans repository includes a number of source files that are not
+covered by the apache license. These files are refered to by licenseinfo.xml
```

Comment:
A very, very minor nit.. I believe that apache should be capitalized to be Apache.

Reply:
+1